tests/src/test_cases/test_image_formats.c

```diff
@@ -92,6 +117,19 @@ void test_image_built_in_decode_recolor(void)
     img_create("XRGB8888", &test_img_cogwheel_xrgb8888, false, true);
     img_create("ARGB8888", &test_img_cogwheel_argb8888, false, true);
 
+#if 0 /*Recolor has issues on draw for now */
```

Comment:
Hi @kisvegabor ,
There's something wrong with img_draw_core function when recolor is enabled. Could you help to check the code? I cannot figure out how to address it for now. 

Thanks!